### PR TITLE
Implement Logger in NodeJS SDK [ch2532]

### PR DESCRIPTION
**Pull request by NehaNaithani**

### CODE REVIEW

File: logger/index.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      ]
+    })
+    this.logger = logger
+  }
+
+  async info (message) {
+    this.logger.info(message)
+  }
+
+  async debug (message) {
```
Comment (Pixcell) [inline logger/index.js]: Why are those methods async ?
I don't understand why logging function return promises

**Review by Pixcell — CHANGES_REQUESTED** [review]

File: logger/index.js (hunk 1 above)
Comment (NehaNaithani) [reply logger/index.js]: removed winston

File: flieLogger/index.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  formatters: {
+    level: (level) => ({ level })
+  }
+}
+
+const timeStamp = () => {
+  return new Date(Date.now()).toISOString()
+}
+
+const dirName = '/var/log/iofog-microservices/'
```
Comment (Pixcell) [inline flieLogger/index.js]: dirname should be configurable, with a default value

**Review by Pixcell — CHANGES_REQUESTED** [review]

File: flieLogger/index.js (hunk 2 above)
Comment (NehaNaithani) [reply flieLogger/index.js]: Updated with dirName configurable

**Review by Pixcell — APPROVED** [review]

**Pull request merged**